src/views/shipments.js

```diff
@@ -336,6 +350,7 @@ export class shipments {
       _id:drug._id,
       generics:drug.generics,
       form:drug.form,
+      brand:drug.brand,
```

Comment:
Let's discuss this one.  I am worried about denormalized data, what happens if someone updates the brand name on the drug page, do all the transactions need to update as well?  But it might be necessary...